superdesk/io/feeding_services/ftp.py

```diff
@@ -197,6 +197,11 @@ def _is_allowed(self, filename, allowed_ext):
         _, ext = os.path.splitext(filename)
         return ext.lower() in allowed_ext
 
+    def _is_empty(self, file_path):
```

Comment:
Does it make sense to put it somewhere else to avoid repeating code? We can create a file mixin and use it for `FileFeedingService` and `FTPFeedingService` or even put it in utils... It's a tiny method, maybe it's not worth it. @petrjasek ?

Reply:
Should we put it to BaseClass `FeedingService`? We intent to do so but not sure it's suitable?!

Reply:
IMO `is_empty` is file related, so the base class is not the right place.
Not a big deal, you can leave it as it is now.